TeachingRecordSystem/src/TeachingRecordSystem.Core/Events/UserDeactivatedEvent.cs

```diff
@@ -5,4 +5,7 @@ namespace TeachingRecordSystem.Core.Events;
 public record UserDeactivatedEvent : EventBase
 {
     public required User User { get; init; }
+    public string? AdditionalReason { get; init; }
+    public string? MoreInformation { get; init; }
```

Comment:
We normally name these like:
```suggestion
    public string? DeactivatedReason { get; init; }
    public string? DeactivatedReasonDetail { get; init; }
```